### PR TITLE
Separate raw audit files from a11y dashboard

### PR DESCRIPTION
**Pull request by rousik**
a11y dashboard does not directly depend on the raw audit files but due to the current directory structure yarn/npm build will have to analyze all those files and that takes forvever.

This PR does the following things:

- dashboard react code is moved into `dashboard/` subdirectory, effectively separating it from raw audit files
- pre-processing script is modified to read the raw audit files outside of dashboard directory
- auto-lighthouse script is modified to write to the location outside of dashboard directory
- supporting scripts were moved from `public/tools` to `scripts` which also excludes them from the build (this is what we want) 
- raw audit file links are using http://raw.githubusercontent.com (i.e. pointing to the github master branch)

### CODE REVIEW

**Comment by Winerip** [comment]
I assume the change in the preprocess script has to do with removing the duplicated files between misc and top?

**Comment by rousik** [comment]
Changes in the pre-process script modify the relative paths where raw audit files are found. The root directory for the node script moved under dashboard/ so we need to access `../public/scan-results`

File: dashboard/package.json [hunk 1]
```diff
@@ -44,9 +44,9 @@
       "last 1 safari version"
     ]
   },
-  "homepage": "https://winerip.github.io/code4good-accessibility",
```
Comment (Winerip) [inline dashboard/package.json:47]: Oops, I thought I was keeping this change out of my pull requests. What does the new value of . Do? Do you have a demo that's published with this? And how long did it take to deploy?

**Comment by Winerip** [comment]
Sorry, comment was unclear. I mean the outputs from that script. It looks like 31 fewer RCB sites were read, that sounds like about how many duplicates there were. I thought that was already addressed though in a separate PR.

File: dashboard/package.json (hunk 1 above)
Comment (rousik) [reply dashboard/package.json:47]: It seems to work, at least locally, and with gh-pages as well. Deployment is quick, say 1 minute or so max.

Comment (rousik) [reply dashboard/package.json:47]: https://rousik.github.io/code4good-accessibility/ has the current deployment of this.

**Comment by rousik** [comment]
> Sorry, comment was unclear. I mean the outputs from that script. It looks like 31 fewer RCB sites were read, that sounds like about how many duplicates there were. I thought that was already addressed though in a separate PR.

Ah. I see. Yes, the duplicates were removed but we did not run the pre-process script when that happened. I have triggered the preprocessing which is finally picking up that change now.

File: dashboard/package.json (hunk 1 above)
Comment (rousik) [reply dashboard/package.json:47]: https://create-react-app.dev/docs/deployment/#serving-the-same-build-from-different-paths provides some explanation for using "." as the value for homepage property.

**Pull request merged**